source/fundamentals/geo.txt

```diff
@@ -0,0 +1,332 @@
+made up of positions.
+
+Here are some common GeoJSON types and how you can specify them with positions:
+
+- ``Point``: a single position. This could represent the location of the
+  MongoDB Headquarters.
+
+  .. code-block:: go
+     :copyable: False
+
+     bson.D{
+         {"name", "HQ"},
+         {"location", bson.D{
+             {"type", "Point"},
+             {"coordinates", []float64{-73.986805, 40.7620853}},
+         }},
+     }
+
+- ``LineString``: an array of two or more positions that forms a series of line
+  segments. This could represent
+  `the route of the Great Wall of China <https://commons.wikimedia.org/wiki/File:GreatWallChina4.png>`__.
+
+  .. code-block:: go
+     :copyable: False
+
+     bson.D{
+         {"name", "Great Wall of China"},
+         {"location", bson.D{
+             {"type", "LineString"},
+             {"coordinates", [][]float64{
+                 {116.572, 40.430},
+                 {116.570, 40.434},
+                 {116.567, 40.436},
+                 {116.566, 40.441},
+             }}},
+         },
+     }
+
+- ``Polygon``: an array of positions in which the first and last
+  position are the same and enclose some space. This could represent
+  `the land within Vatican City <https://commons.wikimedia.org/wiki/File:Vatican_City_map_EN.png>`__.
+
+  .. code-block:: go
+     :copyable: False
+
+     bson.D{
+         {"name", "Vatican City"},
+         {"location", bson.D{
+             {"type", "Polygon"},
+             {"coordinates", [][][]float64{{
+                 {116.572, 40.430},
+                 {116.570, 40.434},
+                 {116.567, 40.436},
+                 {116.572, 40.430},
+             }}},
+         }},
+     }
+
+
+To learn more about the shapes you can use in MongoDB, see the
+:manual:`GeoJSON manual entry </reference/geojson/>`.
+
+For definitive information on GeoJSON, see the
+`official IETF specification <https://datatracker.ietf.org/doc/html/rfc7946>`__.
+
+Legacy Coordinate Pairs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can store geospatial data using ``x`` and ``y`` coordinates on
+a two-dimensional Euclidean plane. We refer to coordinates on a two-dimensional
+plane as "legacy coordinate pairs."
+
+Legacy coordinate pairs have the following structure:
+
+.. code-block:: json
+
+   "<field name>" : [ x, y ]
+
+Your field should contain an array of two values in which the first represents
+the ``x`` axis value and the second represents the ``y`` axis value.
+
+For more information on legacy coordinate pairs, see the
+:manual:`MongoDB server manual page on legacy coordinate pairs </geospatial-queries/#legacy-coordinate-pairs>`.
+
+.. _golang-geospatial-indexes:
+
+Geospatial Indexes
+------------------
+
+To enable querying for geosptial data, you must first create a supporting
+index. The index types that enable geospatial queries are:
+
+
+- ``2dsphere`` for GeoJSON data
+- ``2d`` for legacy coordinate pairs
```

Comment:
Issue: [Avoid using fragments to introduce lists.](https://www.mongodb.com/docs/meta/style-guide/style/lists/)

```suggestion
index. The following index types enable geospatial queries:


- ``2dsphere`` for GeoJSON data
- ``2d`` for legacy coordinate pairs
```


- ``2dsphere`` for GeoJSON data
- ``2d`` for legacy coordinate pairs
```